operator/README.md

```diff
@@ -0,0 +1,220 @@
+# rapidast-operator
+The RapiDAST operator makes it possible to configure RapiDAST scans from your OpenShift or Kubernetes cluster. This allows you to scan your APIs from both inside and outside of your cluster.
+
+## Custom Resource Definitions
+The RapiDAST operator extends the RapiDAST custom resource definition (CRD) 
+
+## Installing the Operator
+
+There are multiple ways to install the operator onto your cluster. Read through, and select the method that best suits your situation.
+
+### Install via Operator Lifecycle Manager (OLM)
+If your cluster has OLM installed, you can use it to install and manage the RapiDAST operator. You can follow the installation instructions [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md). If you're running on an OpenShift cluster, you likely have this installed by default.
+
+Install the operator by adding the following CatalogSource and Subscription resources to your cluster.
+#### Add CatalogSource
+Create a catalog source in a file `catalogsource.yaml` with the following contents. Update the value for namespace as required.
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rapidast-catalog
+  namespace: rapidast
+spec:
+  sourceType: grpc
+  image: quay.io/redhatproductsecurity/rapidast-operator-catalog:v0.0.1
+```
+Apply the resource to your cluster
+```bash
+kubectl apply -f catalogsource.yaml
```

Comment:
an error occurs (apparently because no rapidast namespace exist at this point)

$ k apply -f catalogsource.yaml 
Error from server (NotFound): error when creating "catalogsource.yaml": namespaces "rapidast" not found